### PR TITLE
Loginqueries

### PR DESCRIPTION
**Pull request by rinoma**
Relates #29

### CODE REVIEW

**Review by tyeung2017 — APPROVED** [review]
added login logic, but there are sending token logic yet to be added

File: src/controllers/newuser.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res) => {
+  console.log(req.body);
```
Comment (Karyum) [inline src/controllers/newuser.js:2]: remmember to remove log in the future

**Review by Karyum — APPROVED** [review]

File: src/controllers/newuser.js (hunk 1 above)
Comment (rinoma) [reply src/controllers/newuser.js:2]: we need them!!!!!! to see what's going on
they shall be removed upon completion

Comment (Karyum) [reply src/controllers/newuser.js:2]: That what i **MEAN**

**Pull request merged**